scene/gui/text_edit.cpp

```diff
@@ -30,6 +30,7 @@
 #include "text_edit.h"
 
 #include "message_queue.h"
+#include "modules/gdscript/gdscript_tokenizer.h"
```

Comment:
GDScript is an optional module, we can't have a dependency on it in a core Control.

So this code should be moved somehow to the ScriptEditor, and guarded by `#ifdef GDSCRIPT_ENABLED`.